WordPress/gutenberg
# Button Block Alignment fix

## What?
Fixes https://github.com/WordPress/gutenberg/issues/61085

 This PR resolves an issue with the Button Block alignment and justification options, where the default styles for "Justify items left" and "Align top" were not correctly applied. The fix ensures that the selected options are consistently applied and displayed as expected.

## Why?

This PR is necessary to ensure that users can properly align and justify Button blocks within Gutenberg. The existing issue prevented the correct display of the button alignment and justification styles. When the "Justify items left" and "Align top" options were selected, the expected visual changes were not reflected. This fix improves the user experience by ensuring these options work as intended.

## How?
The issue was fixed by adding custom CSS to target the active state of menu items in the dropdown, specifically for the Button block alignment and justification styles. The is-active class now correctly applies the styles, ensuring that "Justify items left" and "Align top" options are visually reflected in the Button block.

## Testing Instructions
Test the Button Block Alignment and Justification:

- Open a post or page in the Gutenberg editor.
- Add a Button block to the page.
- Change the button’s justification to "Justify items left" and the alignment to "Align Top."
- Ensure that the Button block now displays the selected justification and alignment options correctly.

Check for Visual Changes:

- Confirm that the button is aligned to the left and aligned to the top as selected in the block settings.
- Verify that the styles for both alignment and justification are applied as expected.

## Screenshots or screencast 


https://github.com/user-attachments/assets/33b4f718-d9d6-4d05-8a07-a6c75961fc62
